app/Models/Thread.php

```diff
@@ -88,6 +96,16 @@ public function excerpt(int $limit = 100): string
         return Str::limit(strip_tags(md_to_html($this->body())), $limit);
     }
 
+    public function lastActiveAt(): ?Carbon
+    {
+        return $this->last_active_at;
+    }
+
+    public function touchActivity()
+    {
+        $this->update(['last_active_at' => now()]);
+    }
+
```

Comment:
Instead of doing this through a method on the object, move this into the command itself as:

```php
$reply->last_active_at = now();
$reply->save();
```

Remove `last_active_at` from `$fillable`.

Reply:
Looking at this again, I did it this way as `replies` are polymorphic so, in theory, we may have other models using this relationship in the future.

For that reason, I included the `lastActiveAt` and `touchActivity` methods on the `ReplyAble` interface so any uses of the relation would have to implement these methods too.

Reply:
Then let's do:

```php
$event->reply->replyAble()->last_active_at = now();
$event->reply->replyAble()->save();
```

instead.